### ISSUE TITLE
Expectations Documentation incorrect with css

### ISSUE THREAD

**Issue by weaverryan** [blackfireio/player#21]
Hi guys!

I'm using part of the library and just ran into an issue from the docs: https://blackfire.io/docs/player/expectations. Specifically, examples like this:

```yml
# check that "h1" contains "Welcome"
- css("h1:contains('Welcome')")
```

This is not true, because css() *always* returns a DomCrawler, even if it's empty. So, this expression equates to true. We should probably just remove that from the docs, but the fact that `css()` *always* returns a DomCrawler, even if there's nothing in it, makes the expectations a little trickier to write.

Thanks!

**Comment by fabpot** [comment]
Indeed, fixed in 14ea3f2

The right one is: `css("h1:contains('Welcome')").count() > 0`

**Comment by weaverryan** [comment]
thanks :)
